app/assets/stylesheets/static.scss

```diff
@@ -1,6 +1,7 @@
 $govuk-compatibility-govuktemplate: true;
 $govuk-use-legacy-palette: false;
 
+@import "govuk_publishing_components/component_support";
```

Comment:
Why is this being added to this file?

Reply:
It's because many of the styles imported in this file have been updated to use `govuk-font`  instead of the deprecated `core-x` typography mixins. This imports the govuk mixins via `govuk_publishing_components`

Reply:
Is it in one of the @imports then? Just wanted to check it was needed, as there was no reference to the toolkit typography removed from here.

Reply:
Oh yeah I see what you mean now. Yes so 4 of the `@import "helpers/something"` in this file used to call the deprecated mixin and are now using `govuk-font` instead.

There's an error if I try to remove the component support import.

Reply:
Ah, got it. Thanks for the explanation.